### DISCUSSION TITLE
Duda sobre videos en Pagina Web

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#220] [Q&A] [answered]
Buenas tardes Explorers. Ya voy avanzando y casi terminando  con la Página de los Pasteles. Lamentablemente tengo un pequeño problemita...

Le añadí un archivo multimedia, para darle un poquito de estilo. Al abrirlo de manera local al abrir el archivo index.html no tengo ningún problema y se reproduce bien.

Pero al subirlo a GitHub, eh intentar generar la pagina, me doy cuenta que no tiene reproducción, es decir, no reproduce el video bien, o se traba mucho. A que creen que se deba?

Sera que debo reducir la calidad de video? o subirlo a la nube y cargarlo desde allí.
Aquí mi página web:

https://omarventurap.github.io/Practica-Frontend-Sem2-HTML/index.html

¿Ustedes que sugieren?

**Comment by doguedogue** [comment]
Esta genial tu página, no veo en que pueda ayudarte, mejor ayúdame tu 😁😁😁 🚀

**Reply by OmarVenturaP** [reply]
Jejeje, ya quisiera yo que me ayuden. La verdad soy también novato en este asunto y lo que implemento es lo que veo en tutoriales y sobre todo más intento el famoso "prueba y error". A veces compongo algo y se me descuadra otra cosa, jejeje

**Comment by hackit92** [comment, marked as answer]
Para durar 8 segundos si es un poco pesado, a mi se me reprodujo muy fácil pero es por mi ancho de banda. Hay varios temas a considerar cuando optimizamos multimedia para web, te dejo los puntos en mi opinión:
1. Velocidad de entrega de archivos del servidor (desconozco los recursos que github te permita)
2. Compresión y formato del archivo (al usar video en HTML5 es importante comprimir lo más posible bajando el bitrate y de ser posible también la resolución según el uso que se le dará, al igual que entregar en mp4 con h264, webm y ogv)
3. Ancho de banda del publico objetivo (no será lo mismo lo que puede reproducir una persona con conexión 5g o de fibra óptica que alguien que solo cuenta con adsl o 3g)

Fuera de estos puntos mi recomendación si no quieres ahondar en compresión de video y formatos, utiliza un iframe desde youtube, vimeo o algún otro servicio de reproducción de video (sus algoritmos convierten lo que cargan a los formatos necesarios y los comprimen) Recuerda que no todo es h264. 
PD. Fue muy creativo de tu parte utilizar un fondo de video, es una práctica que yo usaría solo en algunas secciones y acompañado de parallax o quizá en un hero pero me parece muy buena tu implementación =)

**Reply by OmarVenturaP** [reply]
Te agradezco el comentario compañero. Si creeme que pensando en eso tomaré tu recomendación de bajarle el bitrate del vídeo. Y pues si está pesado por eso creo que sea problema. Pero crees que si reproduciendolo desde un iframe, llámese YouTube, no baja la calidad?

**Reply by hackit92** [reply]
Puede que si disminuya la calidad pero en mi experiencia cuando utilizas video para el fondo, no es necesario que tenga una calidad de cine, basta con que se vea fluido y sin el famoso "pixeleado" para que luzca bien. La ventaja de YouTube es que va a darte velocidades superiores al promedio y una de las mejores compresiones posibles, prueba y ve si notas diferencia =)

**Reply by OmarVenturaP** [reply]
Excelente. Gracias compañero. Lo intentaré para mejorar la fluidez de la página.
Muchas gracias por el apoyo.
